morse.go

```diff
@@ -33,24 +33,25 @@ func (h *hacker) Encode(r io.Reader) ([]byte, error) {
 	}
 	data := string(d)
 	data = strings.ToUpper(data)
-	var encodedValue string
+	var encode bytes.Buffer
```

Comment:
change this variable name to `encodedBuffer`or `encodedValue`. `encode` is a verb

Reply:
Ok